libs/gl-testing/gltesting/clients.py

```diff
@@ -74,10 +74,24 @@ def tls(self) -> glclient.TlsConfig:
             .with_ca_certificate(capath.open(mode="r").read())
             .identity(certpath.open(mode="r").read(), keypath.open(mode="r").read())
         )
+    
+    def rune(self) -> str:
+        """Load the devices rune
+        
+        Returns the devices rune if it is available or an empty string
+        if the rune can not be found.
+        """
+        self.log.info("Trying to find a rune")
+        runepath = self.directory / "device.rune"
+        rune = ""
```

Comment:
Empty runes must fail. Especially when we have control over the entire environment which is the case for us in `gl-testing`. If we do not enforce correctness in what we test, then how can we expect our users to?

Reply:
The empty rune is used on registration and recovery, that is why we set it to an empty string here. We might want to add a check if the nobody identity is used and fail early if not and if the rune file can not be found. What do you think?